includes/class-fs-api.php

```diff
@@ -337,6 +362,25 @@ function get( $path = '/', $flush = false, $expiration = WP_FS__TIME_24_HOURS_IN
 			return $cached_result;
 		}
 
+        /**
+         * This method will be removed in the future after migrating Freemius::safe_remote_post() to FS_Api::call().

```

Comment:
I'd propose changing this to:
`@todo Remove this method after migrating Freemius::safe_remote_post() to FS_Api::call().`